app/server.js

```diff
@@ -1,40 +1,59 @@
-'use strict';
-
 const express = require('express');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 const path = require("path");
 const healthzRouter = require("./routes/healthz.router");
-const v1Router = require("./routes/v1.router");
+const productRouter = require("./routes/product.router");
 const fdkExtension = require("./fdk");
+const config = require("./config");
+
 const app = express();
+
+// Middleware to parse cookies with a secret key
 app.use(cookieParser("ext.session"));
+
+// Middleware to parse JSON bodies with a size limit of 2mb
 app.use(bodyParser.json({
     limit: '2mb'
 }));
+
+// Serve environment variables as a JavaScript file
 app.get('/env.js', (req, res) => {
```

Comment:
Is this required @jinalviranii ?

Reply:
Yes this is required to read fp_api_server in frontend component

Reply:
Also if we want static documentation link then we do not need this, but for boilerplate do we need to share how end user can access env on client side? @vivek-gofynd